prow/plugins/sigmention/sigmention_test.go

```diff
@@ -124,7 +124,7 @@ func TestSigMention(t *testing.T) {
 			name:              "Repeat Multiple Sig Labels Different Lines With Other Text",
 			body:              "Code Comment.  Design Review\n@kubernetes/sig-node-proposals\ncc @kubernetes/sig-api-machinery-bugs",
 			expectedRepeats:   []string{"@kubernetes/sig-node-proposals", "@kubernetes/sig-api-machinery-bugs"},
-			expectedNewLabels: []string{"sig/node", "sig/api-machinery", labels.Bug},
+			expectedNewLabels: []string{"kind/design", "sig/node", "sig/api-machinery", labels.Bug},
```

Comment:
For strange reason it got added, I will investigate more -  but it is maybe a side effect of the append before.

Reply:
Alright, this comes from the name of the SIG `sig-node-proposals` that matches the `kindMap` entry "proposals" resulting in label `kind/design` being added at line 128.
Before the change in fakegithub.AddLabel() the label wasn't added because we have f.ExistingLabels != nil and the test only allows to add label to the issue if it already exists on the repo.
I am actually wondering what should be the correct behaviour here?

Reply:
Ah, I was confused about this. I thought `ExistingLabels` were the labels that were preexisting on an issue before the test, but they are actually the set of labels that exist in the repo.  So the change you made to `fakegithub` isn't correct.

Reply:
Ok, added a `LabelsExisting` array for that purpose (sig) and will make a PR to rename `ExistingLabels` to avoid this confusion.

Reply:
#10062